README.md

```diff
@@ -0,0 +1,45 @@
+# khmer
+
+[![Research software impact](http://depsy.org/api/package/pypi/khmer/badge.svg)](http://depsy.org/package/python/khmer)
+[![khmer build status](https://img.shields.io/travis/dib-lab/khmer.svg)](https://travis-ci.org/dib-lab/khmer)
+[![Test coverage](https://img.shields.io/codecov/c/github/dib-lab/khmer.svg)](https://codecov.io/github/dib-lab/khmer)
+[![BSD-3 licensed](https://img.shields.io/badge/license-BSD%203--Clause-blue.svg)](https://github.com/dib-lab/khmer/blob/master/LICENSE)
+
+Welcome to khmer: k-mer counting, filtering and graph traversal FTW!
+
+The official source code repository is at https://github.com/dib-lab/khmer
+and you can read the docs online at http://khmer.readthedocs.io.
+
+There are two mailing lists dedicated to khmer, an announcements-only
+list and a discussion list. To search their archives and sign-up for
+them, please visit the following URLs:
+
+-  Discussion: http://lists.idyll.org/listinfo/khmer
+-  Announcements: http://lists.idyll.org/listinfo/khmer-announce
+
+The maintainers can be contacted at <khmer-project@idyll.org>.
+
+For getting help please see this guide: http://khmer.readthedocs.io/en/stable/user/getting-help.html
+Your first port of call when looking for help should be: https://github.com/dib-lab/khmer/issues
+
+## Important note: cite us!
+
+khmer is *research software*, so you should cite us when you use it
+in scientific publications!  Please see the [CITATION](http://khmer.readthedocs.io/en/stable/citations.html)
+file for citation information.
+
+
+## Quick install
+
+```
+pip install khmer
```

Comment:
That gets a lot of attention in #1529. If people are interested in "the right way" to do things, they should look at the instructions anyway. If they're impatient the virtualenv stuff is just noise.